sw/device/tock/tock/Cargo.toml

```diff
@@ -0,0 +1 @@
+/tanap1/proj_cd14/opentitan/jlevy/ot_pr/opentitan/sw/device/tock/tock/Cargo_remote.toml
```

Comment:
This file should not be edited. It is this line here that leads to the CI failure during the documentation build check.